pinot-common/src/main/java/org/apache/pinot/common/config/SegmentsValidationAndRetentionConfig.java

```diff
@@ -55,7 +57,7 @@
   private String timeColumnName;
 
   @ConfigKey(value = "timeType")
-  private String timeType;
+  private TimeUnit _timeType;
```

Comment:
Is this compatible with the zookeeper config we store? Will reads work right? Please test manually that the reads of old configs can work correctly.

Reply:
Tested (we have this covered in TableConfigTest)